test/public/runs/runsPerDataPass.overview.test.js

```diff
@@ -68,14 +68,17 @@ const DETECTORS = [
  * @param {number} params.dataPassId id of data pass on Data Passes per LHC Period page
  * @return {Promise<void>} promise
  */
-const navigateToRunsPerDataPass = async (page, { lhcPeriodId, dataPassId }) => {
+const navigateToRunsPerDataPass = async (page, { lhcPeriodId, dataPassId, rowsCount }) => {
```

Comment:
what is this parameter? Documentation
rowsCount do not belongs to `params` in my opinion, it should be another parameter, and it should be clear that you wait for this (like `expectedRowsCount`)

Reply:
applied